.pylintrc

```diff
@@ -27,7 +27,7 @@ argument-rgx=[a-z_][a-z0-9_]{0,30}$
 
 [FORMAT]
 
-max-line-length=79
+max-line-length=160
```

Comment:
There was quite a consensus in the IRC meeting on that topic yesterday. @gundalow what should we do?

Reply:
Personally I think that 160 is too long. Everyone is going to have their preferences but the fact that yapf would break git blame is imo a huge indicator that this isn't worth it.

Just my two cents

Reply:
OTOH I think that good thing to add is `.editorconfig` so that if users enable the plugin in their editor it would help them writing code closer to the standard.

I also don't like having an automatic formatter for a list of reasons but we'll get to it another time...
https://twitter.com/dabeaz/status/1070256908257378306?s=19

Reply:
OK, I'm pulling rank on this.
We (and upstream ansible-lint) are moving to 160 characters `</topic>`
Once ansible-lint has been updated we can remove that line of config

Reply:
From that meeting log it looks like nobody cares. Agreed to whatever suggestion was proposed. I provided a set of supporting arguments at least..
I feel really bad about breaking the history.
Is there anybody who is going to write a script which would split gigantic format change which yapf it going to generate to the right users?
Also, why do you underestimate huge accessibility problem?

Reply:
OK, let's discuss this more, maybe there is an impact we hadn't considered.

I'll accept responsibility on this

Reply:
For the record, I do care and having to go through 2 rounds of `git blame` instead of one is not a good excuse in my opinion. Also calling it "looks like nobody cares" if several (6 or so) people say "+1" or something similar is a bit dishonest in my opinion.

I also like the fact that a consistent coding style is enforced by a tool instead of a maintainer. If someone doesn't like to view code with 160 character lines, they can configure their editor to reformat the code any way they like locally.

Reply:
Please use the same standard for all Ansible projects.
Or like @mpdehaan used to say: *If 160 is too long, buy a wider screen.*

Reply:
Merge this please.